src/DependenciesPackager/Program.cs

```diff
@@ -184,7 +189,7 @@ static int Main(string[] args)
                 packagesVersion,
                 useCli,
                 quiet,
-                runtimes,
+                runtime,
```

Comment:
Settled with @glennc, we will break the windows package cache into two zip packages x64 vs x86. Therefore I'll keep my change.